### PR TITLE
feat(collector-options-form): modify loading design

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

image

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
l
G
t
M

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue [hunk 1]
```diff
@@ -49,6 +50,17 @@
...
+                <div class="loading-box">
+                    <div class="loading-spinner">
+                        <p-spinner size="xl" />
+                    </div>
+                    <div class="loading-description">
+                        <p>{{ $t('INVENTORY.COLLECTOR.CREATE.LOADING_DESC1') }}</p>
+                        <p>{{ $t('INVENTORY.COLLECTOR.CREATE.LOADING_DESC2') }}</p>
+                    </div>
+                </div>
+            </template>
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue:63]: praise: 👍

**Review by yuda110 — APPROVED** [review]
LGTM!

**Review by WANZARGEN — APPROVED** [review]
LGTM!!!

**Pull request merged**